tests/starwars/schema.py

```diff
@@ -1,24 +1,25 @@
 from graphql.type import (GraphQLArgument, GraphQLEnumType, GraphQLEnumValue,
                           GraphQLField, GraphQLInterfaceType, GraphQLList,
                           GraphQLNonNull, GraphQLObjectType, GraphQLSchema,
-                          GraphQLString, GraphQLBoolean)
+                          GraphQLString, GraphQLBoolean, GraphQLInt,
+                          GraphQLInputObjectType, GraphQLInputObjectField)
 
-from .fixtures import getCharacters, getDroid, getFriends, getHero, getHuman, updateHumanAlive
+from .fixtures import createReview, getCharacters, getDroid, getFriends, getHero, getHuman, reviewAdded
 
 episodeEnum = GraphQLEnumType(
     'Episode',
     description='One of the films in the Star Wars Trilogy',
     values={
         'NEWHOPE': GraphQLEnumValue(
-            4,
+            value='NEWHOPE',
```

Comment:
Is there a reason why you changed the values? When they are different, the tests can detect better when the wrong thing is returned (name instead of value) is returned. Btw, these numeric values are also used by the GraphQL.js and graphq-core tests.

Reply:
> Based on this graphene PR and reading about GraphQL sample schema, I tried to replicate the subscription feature using gql library.

I thought they used string values for the enumerable, as seen [here](https://github.com/apollographql/starwars-server/blob/master/data/swapiSchema.js#L361).

Reply:
They construct a schema from SDL, so they can't define values. In that case, the values are the names. But since we have a schema with server side values, I think we should also use them. The schema in GraphQL.js that uses the numeric values is [here](https://github.com/graphql/graphql-js/blob/master/src/__tests__/starWarsSchema.js).

Reply:
Okay, I will bring back the old values instead of using string as currently does.